build/Packages.props

```diff
@@ -0,0 +1,61 @@
+<Project>
+
+<PropertyGroup>
+    <NuGetPackageVersion>4.8.0-preview1.5185</NuGetPackageVersion>
+
+    <!-- Prodcon-overrideable, using repo-toolset naming convention -->
+    <NuGetBuildTasksVersion>$(NuGetPackageVersion)</NuGetBuildTasksVersion>
+    <NuGetCommandsVersion>$(NuGetPackageVersion)</NuGetCommandsVersion>
+    <NuGetProtocolVersion>$(NuGetPackageVersion)</NuGetProtocolVersion>
+</PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Update="LargeAddressAware" Version="1.0.3" />
+    <PackageReference Update="Microsoft.Build.Build" Version="15.5.180" />
+    <PackageReference Update="Microsoft.Build.Framework" Version="15.5.180" />
+    <PackageReference Update="Microsoft.Build.NuGetSdkResolver" Version="$(NuGetBuildTasksVersion)" />
+    <PackageReference Update="Microsoft.Build.Utilities.Core" Version="15.5.180" />
+    <PackageReference Update="Microsoft.CodeAnalysis.Build.Tasks" Version="3.0.0-beta1-61516-01" />
+    <PackageReference Update="Microsoft.DotNet.BuildTools.GenAPI" Version="2.1.0-prerelease-02404-02" />
+    <PackageReference Update="Microsoft.Extensions.DependencyModel" Version="2.1.0" />
+    <PackageReference Update="Microsoft.Net.Compilers" Version="$(MicrosoftNetCompilersVersion)" />
+    <PackageReference Update="Microsoft.NETCore.Compilers" Version="$(MicrosoftNetCoreCompilersVersion)" />
```

Comment:
Do you use Microsoft.Net.Compiler and Microsoft.NETCore.Compilers directly in the repo? The toolset adds these automatically.

Reply:
I just moved it, didn't add it new. So not sure.

Reply:
IIRC we use them to deploy binaries into our bootstrap folder. But maybe that stuff is now copied from the LKG instead.